Makefile

```diff
@@ -322,6 +322,31 @@ generate.crd-kustomize:
 generate.api: controller-gen
 	$(CONTROLLER_GEN) object:headerFile="hack/generators/boilerplate.go.txt" paths="./$(API_DIR)/..."
 
+# Alias for consistency with kubernetes-configuration naming
+.PHONY: generate.deepcopy
+generate.deepcopy: generate.api
+
+.PHONY: generate.apitypes-funcs
+generate.apitypes-funcs:
+	go run ./scripts/apitypes-funcs
+
+.PHONY: generate.docs
+generate.docs: generate.apidocs
+
+.PHONY: generate.apidocs
+generate.apidocs: crd-ref-docs
+	./scripts/apidocs-gen/generate.sh $(CRD_REF_DOCS)
+
+# Apply same auto-fixes as golangci-lint to keep generate and lint consistent
+.PHONY: generate.lint-fix
+generate.lint-fix: golangci-lint
```

Comment:
Is this required as we have `lint` target in makefile?

Reply:
Yes. I added the `--fix` here https://github.com/Kong/kong-operator/pull/2342/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R343